when-shit-hits-the-fan.adoc

```diff
@@ -641,23 +640,23 @@ the next section.
 image::bip66-timeline-2.png[{big-width}]
 
 A key takeaway from this is that BIPs should be made more or less
-_atomic_; complete enough to provide something useful or solve a
+_atomically_, meaning that they should be complete enough to provide something useful or solve a
```

Comment:
I'd say we go back to `_atomic_`. As in "the BIP should be atomic"

Reply:
Then I would remove "made", what do you think?

Reply:
Yeah, that's a good idea!